lib/presentation/mixins/connect_page_mixin.dart

```diff
@@ -106,7 +112,8 @@ mixin ConnectPageMixin {
       redirectUrl: redirectUrl,
     );
     final urlScheme = _getRedirectUrlScheme(redirectUrl);
-    return await FlutterWebAuth2.authenticate(
+
+    return FlutterWebAuth2.authenticate(
```

Comment:
why dont you remove await

Reply:
Since `authenticateWithWebAuth` and `authenticate` both return `Future<String>` it's the same I think. But I will revert this change since it's not relevant anymore.